LICENSE

```diff
@@ -20,3 +22,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
```

Comment:
Out of curiosity, did you check to see if the rest of the wording is the same between the current License and the doc? That might be good to validate.

Reply:
they're the same